.github/workflows/run_periodic_tests.yml

```diff
@@ -114,42 +114,54 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: python -m nox -s scripts
 
-  #M-series Mac Mini
+  # M-series Mac Mini
   build-apple-mseries:
     if: github.repository_owner == 'pybamm-team'
     needs: style
     runs-on: [self-hosted, macOS, ARM64]
     env:
       GITHUB_PATH: ${PYENV_ROOT/bin:$PATH}
+      LD_LIBRARY_PATH: $HOME/.local/lib
     strategy:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
 
     steps:
       - uses: actions/checkout@v4
-      - name: Install python & create virtualenv
+      - name: Install Python & create virtualenv
         shell: bash
         run: |
           eval "$(pyenv init -)"
           pyenv install ${{ matrix.python-version }} -s
           pyenv virtualenv ${{ matrix.python-version }} pybamm-${{ matrix.python-version }}
 
-      - name: Install dependencies & run unit tests for Windows and MacOS
+      - name: Install build-time dependencies & run unit tests for M-series macOS runner
```

Comment:
Are we not running it for windows now?

Reply:
This is a separate job for just the M-series runner, the job step was written wrongly :)